.github/workflows/release.yml

```diff
@@ -61,6 +61,12 @@ jobs:
       - name: Build ⚒️
         run: bunx @vscode/vsce package
 
+      - name: Attest artifacts ✅
+        uses: actions/attest@7305951e905fb742188aa16c1d23409b13565e26 # v1.3.3
+        # Read: https://docs.github.com/en/actions/security-guides/using-artifact-attestations-to-establish-provenance-for-builds
```

Comment:
The article mentions the need of adding another permission:
```yaml
permissions:
  id-token: write
  contents: read
  attestations: write <--- Add write permissions for attestations
```

Do we need this?

Reply:
Exactly, I forgot. This is really necessary

Reply:
Exactly, I forgot. This is really necessary

Reply:
Done